docs/contributing/community.md

```diff
@@ -38,7 +38,7 @@ The Gatsby community welcomes contributions. Please refer to the guides below on
 
 Sign up for the Gatsby newsletter to keep up with the latest from the Gatsby community! Hear about new features, tips & tricks, and what people are building.
 
-[Sign up here](/newsletter/)
+<EmailCaptureForm signupMessage="Want to keep up with the latest tips &amp; tricks? Subscribe to our newsletter!" />
```

Comment:
We'd also need to get rid of `www/src/pages/newsletter.js`.

Reply:
should I remove that file?

Reply:
yup!

Reply:
done